src/content/documentation/develop/differences-between-desktop-and-android-extensions.md

```diff
@@ -72,49 +66,54 @@ A detailed list of the WebExtension APIs supported in Firefox for Android is pro
-- [`browsingData`](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/browsingData), which means you cannot offer users features to clear browser data such as history, downloads, passwords, and alike.
 - The [`chrome_url_overrides`](https://developer.mozilla.org/Add-ons/WebExtensions/manifest.json/chrome_url_overrides) and [`chrome_settings_overrides`](https://developer.mozilla.org/Add-ons/WebExtensions/manifest.json/chrome_settings_overrides) `manifest.json` keys, which means you cannot add custom home and new tab pages.
-- [`contextMenus`](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/contextMenus), which means you cannot add options to context menus.
+- [`menus`](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/menus), which means you cannot add options to context menus.
 - [`history`](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/history), which means you cannot search or manipulate the history of browsed pages.
 - [`omnibox`](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/omnibox) and the related [`omnibox`](https://developer.mozilla.org/Add-ons/WebExtensions/manifest.json/omnibox) `manifest.json` key, which means you cannot provide custom address bar suggestions.
 - [`sessions`](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/sessions), which means you cannot list and restore tabs that have been closed while the browser has been running.
-- The [`options_ui`](https://developer.mozilla.org/Add-ons/WebExtensions/manifest.json/options_ui) `manifest.json` key can be used only from Firefox for Android version 57 and above.
 
 Developer tools for Firefox for Android are provided through remote debugging mechanisms [over USB](https://developer.android.com/studio/debug/dev-options) or [Wi-Fi](https://developer.mozilla.org/docs/Tools/Remote_Debugging/Debugging_Firefox_for_Android_over_Wifi) that connect to the WebIDE on a desktop. Therefore, Firefox for Android does not provide any built-in developer tools and its extensions do not support the APIs to extend the developer tools:
 
 - [`devtools.inspectedWindow`](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/devtools.inspectedWindow)
 - [`devtools.network`](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/devtools.network)
 - [`devtools.panels`](https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/devtools.panels) and the related [`devtools_page`](https://developer.mozilla.org/Add-ons/WebExtensions/manifest.json/devtools_page) `manifest.json` key.
 
+### Effect on your add-on UI
+
+You cannot expose your add-on through a sidebar or context menu. However, you can expose your extension as a toolbar or address bar button.
+
+Depending on the manifest version your app is using you add a toolbar button using:
```

Comment:
@mixedpuppy where does this display, presumably in the browser menu?

Reply:
**note:** @rebloor the browserAction/pageAction button will be accessible through the hamburger menu of the Firefox for Android app, nested inside the "Add-ons" sub panel.

Follows (inside the collapsable details below) a couple of screenshot to show what I meant with what I described above.

<details>

![Screenshot from 2023-07-14 19-10-23](https://github.com/mozilla/extension-workshop/assets/11484/bb8a180e-7f9e-4255-bf43-d7ae36c15377)

![Screenshot from 2023-07-14 19-10-53](https://github.com/mozilla/extension-workshop/assets/11484/46132495-f97b-4ce8-a511-0a63183d7cd5)

</details>

Reply:
Thanks @rpl I have updated the description.